1,たまにIDがとれない


Traceback (most recent call last):
  File "main_bk.py", line 73, in <module>
    orderInstance = order(mode)
  File "main_bk.py", line 43, in order
    print "orderInstance.id = %s" % orderInstance.getId()
  File "/home/ec2-user/work/oanda_dev/work/order_info.py", line 14, in getId
    return int(self.id)
TypeError: int() argument must be a string or a number, not 'NoneType'

2, 決済した後の回帰分析で謎のエラーが起きる
